Tasks/Common/azure-arm-rest/azure-arm-app-service-kudu.ts

```diff
@@ -0,0 +1,305 @@
+import msRestAzure = require('./azure-arm-common');
+import tl = require('vsts-task-lib/task');
+import util = require('util');
+import webClient = require('./webClient');
+import Q = require('q');
+import { ToError } from './AzureServiceClient';
+import { WebJob, SiteExtension } from './azureModels';
+
+export class KuduServiceManagementClient {
+    private _scmUri;
+    private _accesssToken: string;
+
+    constructor(scmUri: string, accessToken: string) {
+        this._accesssToken = accessToken;
+        this._scmUri = scmUri;
+    }
+
+    public async beginRequest(request: webClient.WebRequest): Promise<webClient.WebResponse> {
+        request.headers = request.headers || {};
+        request.headers["Authorization"] = "Basic " + this._accesssToken;
+        request.headers['Content-Type'] = 'application/json; charset=utf-8';
+        var httpResponse = webClient.sendRequest(request);
+        return httpResponse;
+    }
+
+    public getRequestUri(uriFormat: string) {
+        uriFormat = uriFormat[0] == "/" ? uriFormat : "/" + uriFormat;
+        return this._scmUri + uriFormat;
+    }
+}
+
+export class Kudu {
+    private _client: KuduServiceManagementClient;
+    private _scmUri: string;
```

Comment:
You can remove _scmUri as it's already there with KuduServiceManagementClient

Reply:
Done. thanks